README.md

```diff
@@ -32,4 +32,4 @@ if __name__ == '__main__':
 
 ```
 
-As long as you get an instance of `krux_boto.boto.Boto3`, the rest are the same. Refer to `krux_boto` module's [README](../krux_boto/README.md) on various ways to instanciate the class.
+As long as you get an instance of `krux_boto.boto.Boto3`, the rest are the same. Refer to `krux_boto` module's [README](https://github.com/krux/python-krux-boto/blob/master/README.md) on various ways to instanciate the class.
```

Comment:
Yeah, I would've preferred a relative links, but I don't think I can go into a different repo using relative links. The only way that I can get working is very hacky.